src/elements/content-sidebar/activity-feed/activity-feed/ActivityThread.js

```diff
@@ -62,16 +64,44 @@ const ActivityThread = ({
     repliesTotalCount = 0,
     translations,
 }: Props) => {
-    const [isExpanded, setIsExpanded] = React.useState(isAlwaysExpanded);
+    const { length: repliesLength } = replies;
```

Comment:
can replies length be 0?

Reply:
That is correct, when there are no replies the length is 0. Theoretically `onHideRepliesHandler` shouldn't be called when `repliesLength = 0,` but I've added additional check in `onHideRepliesHandler`.